build.gradle

```diff
@@ -166,6 +166,7 @@ dependencies {
     compile group: 'net.sourceforge.argparse4j', name: 'argparse4j', version: '0.7.0'
     compile group: 'net.sourceforge.pcgen', name: 'PCGen-base', version:'1.0.55'
     compile group: 'net.sourceforge.pcgen', name: 'PCGen-Formula', version:'1.0.64'
+    compile group: 'org.jetbrains', name: 'annotations', version:'15.0'
```

Comment:
Its the same thing AFAIK. You just need a place to get the annotations from.
You can also get the same annotations from com.google.code.findbugs